python/lsst/obs/base/defineVisits.py

```diff
@@ -57,6 +59,62 @@
 from ._instrument import loadCamera
 
 
+class VisitSystem(enum.IntEnum):
```

Comment:
Does this actually need to be an `IntEnum` rather than just an `Enum` with `int` values?  From the [Python docs](https://docs.python.org/3/library/enum.html#derived-enumerations):

> For the majority of new code, [Enum](https://docs.python.org/3/library/enum.html#enum.Enum) and [Flag](https://docs.python.org/3/library/enum.html#enum.Flag) are strongly recommended, since [IntEnum](https://docs.python.org/3/library/enum.html#enum.IntEnum) and [IntFlag](https://docs.python.org/3/library/enum.html#enum.IntFlag) break some semantic promises of an enumeration (by being comparable to integers, and thus by transitivity to other unrelated enumerations). [IntEnum](https://docs.python.org/3/library/enum.html#enum.IntEnum) and [IntFlag](https://docs.python.org/3/library/enum.html#enum.IntFlag) should be used only in cases where [Enum](https://docs.python.org/3/library/enum.html#enum.Enum) and [Flag](https://docs.python.org/3/library/enum.html#enum.Flag) will not do; for example, when integer constants are replaced with enumerations, or for interoperability with other systems.

Reply:
I cargo-culted from an example I had lying around.